CleverCloud/clever-components
# feat(cc-pricing-product): add translations for `dedicated` and `is-migratable` features

Fixes #804

## How to review?

- check the commit (from GitHub should be just fine),
- check the [`cc-pricing-product - Data loaded - (fake product)` story](https://clever-components-preview.cellar-c2.services.clever-cloud.com/cc-pricing-product/add-missing-feature-translations/index.html?path=/story/%F0%9F%9B%A0-pricing-cc-pricing-product--data-loaded-with-fake-product) in both English and French. The first product table should show the following categories:
  -  "Migration tool" / "Outil de migration" (`boolean` type)
  -  "Dedicated" / "Dédié" (`boolean` type)
